development/spellCheckerSkipWords.js

```
@@ -823,6 +823,7 @@ module.exports = [
   'Blockflow',
   'fet',
   'cex',
+  'Lokalise',
   'bundler',
-  'dai',
+  'dai'
```

ESLint

[error] Insert `,`

(prettier/prettier)